boards/adafruit-pybadge/Makefile.dep

```diff
@@ -0,0 +1,16 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+ifneq (,$(filter disp_dev,$(USEMODULE)))
+  # Driver not supported yet
+  # USEMODULE += st7735
+endif
+
+ifneq (,$(filter mtd,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_spi_on_qspi
+  USEMODULE += mtd_spi_nor
+endif
```

Comment:
Do you know if it comes with a filesystem by default?

Reply:
No idea, and if it does, I probably broke it when I ran `tests/mtd` auto tests.

Reply:
Given that the default bootloader [works by exposing a USB block device](https://learn.adafruit.com/adafruit-pybadge/updating-the-bootloader), it could be assumed that what is exposed is the attached flash memory, and is FAT formatted. (Could still be partitioned, though).

Do we need a run-me-first-on-a-new-board test that dumps all the content (full flash memory to backup the vendor bootloader; any external flash) to answer such questions in the future?

Reply:
The bootloader exposes the *internal* flash